spec/paperclip/attachment_spec.rb

```diff
@@ -34,9 +34,9 @@
   it "does not delete styles that don't get reprocessed" do
     file = File.new(fixture_file("50x50.png"), 'rb')
     rebuild_class styles: {
-      small: "100x>",
-      large: "500x>",
-      original: "42x42#"
+      small: '100x>',
+      large: '500x>',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.